charts/node-local-dns/values.yaml

```diff
@@ -73,3 +73,19 @@ config:
   # Could be retrieved with `kubectl get svc kube-dns -n kube-system -o jsonpath={.spec.clusterIP}`
   dnsServer: ""
   clusterDomain: cluster.local
+
+serviceMonitor:
+  # -- If true, a ServiceMonitor CRD is created for a prometheus operator.
+  # https://github.com/coreos/prometheus-operator
+  #
+  enabled: false
+  # -- HTTP scheme to use for scraping.
+  scheme: http
```

Comment:
I guess it is fine, but I'm not sure CoreDNS is able to serve https on its own.

Reply:
?

Reply:
I was not sure this value was useful since https would not work, since coreDNS will serve only on http, and there is no kube-rbac-proxy here.